PC/launcher2.c

```diff
@@ -2052,7 +2054,8 @@ struct StoreSearchInfo {
 
 
 struct StoreSearchInfo STORE_SEARCH[] = {
-    { L"3", /* 3.11 */ L"9NRWMJP3717K" },
+    { L"3", /* 3.12 */ L"9NCVDN91XZQP" },
```

Comment:
Should this be part of the release manager's playbook when making a 3.x.0 release?

Reply:
Not a bad idea, though it's in a bit of a weird spot really. The most likely way for people to get the update with the change is to install the new release, at which point they won't hit this code path (or in fact any Python 3 release, so even updating an old one).

It's possible to install the launcher separately with winget (or directly if you know the URL), which is what makes this feature at all useful, but those updates will lag anyway as we don't control them ourselves.

Plus it's all a bit of an experiment, so I kinda want to see how it goes before baking it into our formal process.